ui/app/templates/components/masked-input.hbs

```diff
@@ -0,0 +1,21 @@
+<div class="masked-input {{if shouldObscure "masked"}} {{if displayOnly "display-only"}}" data-test-masked-input>
+	{{#if displayOnly}}
+		<pre class="masked-value display-only">{{displayValue}}</pre>
+	{{else}}
+  <textarea
+    class="input masked-value"
+    rows=1
+    wrap="off"
+    placeholder="value"
+    onfocus={{action "setFocus" true}}
+    onblur={{action "setFocus" false}}
+    onkeydown={{action onKeyDown}}
+    onchange={{action updateValue}}
```

Comment:
I think the `action` helpers here are unneeded, as these attributes are already bound closure actions.